packages/rspack/src/global.d.ts

```diff
@@ -109,3 +109,55 @@ declare module "neo-async" {
 	export const forEach: typeof each;
 	export const forEachLimit: typeof eachLimit;
 }
+
+type RuntimeGlobalsType = {
```

Comment:
Why adding this manually?

Reply:
I tried exporting via runtime_globals.rs, but it failed. Can you guide me? Thanks

Reply:
Exporting from rust will create a huge runtime overhead. Please stick to JS impl.

Reply:
This type can be removed directly.

Reply:
i understand ，thank